src/structures/vroom/input/input.cpp

```diff
@@ -57,6 +59,7 @@ void Input::set_geometry(bool geometry) {
   _geometry = geometry;
 }
 
+#if USE_ROUTING
 void Input::add_routing_wrapper(const std::string& profile) {
```

Comment:
The `add_routing_wrapper` definition should also be conditioned by `USE_ROUTING`.